torch/testing/_internal/common_utils.py

```diff
@@ -408,33 +421,28 @@ def wrapper(*args, **kwargs):
 def wrapDeterministicFlagAPITest(fn):
```

Comment:
Please elaborate in this comment that tests using this wrapper need to start a subprocess (how and why) so that their cuBLAS is initialized with the new workspace config

Reply:
Added, let me know what you think